lib/theme.json

```diff
@@ -273,6 +273,12 @@
 					"radius": true
 				}
 			},
+			"core/image": {
+				"lightbox": {
+					"enabled": null,
```

Comment:
I set the `lightbox.enabled` attribute to NULL by default so that we can determine whether the Behaviors syntax should override it or not. If the values can be migrated automatically with the `theme.json` migration, then we remove this.

Reply:
I removed it in 6459d567adc9114e2b228de0e0b4ee7bf0f0a1e1